modules/rosa-sts-creating-roles.adoc

```diff
@@ -0,0 +1,616 @@
+                 "ec2:DescribePrefixLists",
+                 "ec2:DescribeRegions",
+                 "ec2:DescribeReservedInstancesOfferings",
+                 "ec2:DescribeRouteTables",
+                 "ec2:DescribeSecurityGroups",
+                 "ec2:DescribeSubnets",
+                 "ec2:DescribeTags",
+                 "ec2:DescribeVolumes",
+                 "ec2:DescribeVpcAttribute",
+                 "ec2:DescribeVpcClassicLink",
+                 "ec2:DescribeVpcClassicLinkDnsSupport",
+                 "ec2:DescribeVpcEndpoints",
+                 "ec2:DescribeVpcs",
+                 "ec2:DetachInternetGateway",
+                 "ec2:DisassociateRouteTable",
+                 "ec2:GetEbsDefaultKmsKeyId",
+                 "ec2:ModifyInstanceAttribute",
+                 "ec2:ModifyNetworkInterfaceAttribute",
+                 "ec2:ModifySubnetAttribute",
+                 "ec2:ModifyVpcAttribute",
+                 "ec2:ReleaseAddress",
+                 "ec2:ReplaceRouteTableAssociation",
+                 "ec2:RevokeSecurityGroupEgress",
+                 "ec2:RevokeSecurityGroupIngress",
+                 "ec2:RunInstances",
+                 "ec2:TerminateInstances",
+                 "elasticloadbalancing:AddTags",
+                 "elasticloadbalancing:ApplySecurityGroupsToLoadBalancer",
+                 "elasticloadbalancing:AttachLoadBalancerToSubnets",
+                 "elasticloadbalancing:ConfigureHealthCheck",
+                 "elasticloadbalancing:CreateListener",
+                 "elasticloadbalancing:CreateLoadBalancer",
+                 "elasticloadbalancing:CreateLoadBalancerListeners",
+                 "elasticloadbalancing:CreateTargetGroup",
+                 "elasticloadbalancing:DeleteLoadBalancer",
+                 "elasticloadbalancing:DeleteTargetGroup",
+                 "elasticloadbalancing:DeregisterInstancesFromLoadBalancer",
+                 "elasticloadbalancing:DeregisterTargets",
+                 "elasticloadbalancing:DescribeInstanceHealth",
+                 "elasticloadbalancing:DescribeListeners",
+                 "elasticloadbalancing:DescribeLoadBalancerAttributes",
+                 "elasticloadbalancing:DescribeLoadBalancers",
+                 "elasticloadbalancing:DescribeTags",
+                 "elasticloadbalancing:DescribeTargetGroupAttributes",
+                 "elasticloadbalancing:DescribeTargetGroups",
+                 "elasticloadbalancing:DescribeTargetHealth",
+                 "elasticloadbalancing:ModifyLoadBalancerAttributes",
+                 "elasticloadbalancing:ModifyTargetGroup",
+                 "elasticloadbalancing:ModifyTargetGroupAttributes",
+                 "elasticloadbalancing:RegisterInstancesWithLoadBalancer",
+                 "elasticloadbalancing:RegisterTargets",
+                 "elasticloadbalancing:SetLoadBalancerPoliciesOfListener",
+                 "iam:AddRoleToInstanceProfile",
+                 "iam:GetInstanceProfile",
```

Comment:
Add `"iam:DeleteInstanceProfile",`

Reply:
Installer will throw a warning without this and leave around a bootstrapping instance profile.